package.json

```diff
@@ -163,9 +163,9 @@
     "css-loader": "^2.1.1",
     "detect-port": "^1.3.0",
     "devtron": "^1.4.0",
-    "electron": "6.1.7",
-    "electron-builder": "^20.38.5",
-    "electron-debug": "^2.1.0",
+    "electron": "7.1.11",
```

Comment:
do you think you can upgrade to 7.1.14 or to the latest version of 7x branch?

Reply:
Yes, I don't see why not.
(Tested with 7.1.11, wish to upgrade to version 9 if this branch gets merged).